app/views/layouts/application.html.erb

```diff
@@ -25,7 +26,10 @@
         <ul class="nav navbar-nav">
           <!-- TODO: Make tab active when page is active. -->
           <li class="<%= 'active' if current_page?(root_path) %>"><%= link_to "Welcome", root_path %></li>
-          <li><%= link_to "Admin", "/admin" %></li>
+          <% if current_user.admin? %>
```

Comment:
We're gonna need to check `if current_user && current_user.admin` otherwise it'll crash when we're not signed in.